src/main/frontend/handler/ui.cljs

```diff
@@ -16,14 +16,36 @@
             [frontend.mobile.util :as mobile]
             [electron.ipc :as ipc]))
 
+(defn- get-css-var-value
+  [var-name]
+  (.getPropertyValue (js/getComputedStyle (.-documentElement js/document)) var-name))
+
 ;; sidebars
+(defn- get-right-sidebar-width
+  []
+  (or (.. (js/document.getElementById "right-sidebar") -style -width)
+      (get-css-var-value "--right-sidebar-width")))
+
+(defn- persist-right-sidebar-width!
+  []
+  (storage/set "ls-right-sidebar-width" (get-right-sidebar-width)))
```

Comment:
Might better store the width here, because the width will not be persisted if user closes the app directly. https://github.com/logseq/logseq/blob/e0888b896f525465b99dff9c28806d8001fc2d37/src/main/frontend/components/right_sidebar.cljs#L188

Reply:
I was also thinking about putting "persist" here, but I was a little worried if writing to local storage while dragging is too slow. 
But according to some trivial search according to https://stackoverflow.com/questions/2010892/how-to-store-objects-in-html5-localstorage, it seems it is fast enough. I will move the persist fn here in a new commit.